docs/platform/README.md

```diff
@@ -0,0 +1,49 @@
+# Serverless Platform (Beta)
+
+The Serverless Platform is currently in experimental beta. If you'd like to participate in the beta, simply follow the instructions below.
```

Comment:
Could we provide a link to a Platform "about" page here so that users can take a look what this "Platform" is all about?